Domain/Models/SeaRoutesResponse.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.Json.Serialization;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+
+namespace Domain.Models
+{
+    public class Routepoint
+    {
+
+        [JsonProperty("lon")]
```

Comment:
Why jsonproperty is required?   If  it is used for writing props in camelcasing  , then  using json settings/configuration  and passing it to the serializer function would be cleaner approach. Hence you won't need to repeat attribute for all the properties.